Add Wiley search example

haruanm opened the pull request:
*Issue #, if available:*

*Description of changes:*
This PR adds requests to the Wiley API to search for health articles.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

haruanm commented:
@hasanp87 could you check our current implementation?

hasanp87 commented:
@haruanm thanks for your contribution. Can you address the below questions : 
1. There are a few binary files in the commit, can you remove
2. The readme calls out deployment via CFN, however, it seems that the python sdk agent creation is required - can you clarify in the README . We will test deploy at our end once you do so. 
3. Can you include a few sample questions (perhaps in a notebook format or in README) to show how to interact with the agent
4. Note the HCLS sample agents in this repo are under MIT-0 license,

hasanp87 reviewed (changes requested):
pls see comment in the PR

haruanm commented:
Hey @hasanp87 I have changed the Readme to explain that there are 2 deployment options, 1 with python and 1 using CFN, and also removed the pyc files.

I still don't have question examples, do you mind if we add that on a second PR?
